### ISSUE TITLE
Negative paddings

### ISSUE THREAD

**Issue by jojaii** [onnx/keras-onnx#146]
The converter is outputting negative paddings in the graph whenever "same" padding is encountered. The model I used is the minixception model (accessible [here](https://github.com/oarriaga/face_classification/blob/master/trained_models/emotion_models/fer2013_mini_XCEPTION.107-0.66.hdf5)).

This can be seen in the "conv2d_3" layer in the model. You may download the file [here](https://drive.google.com/file/d/1tJrFuYGVMwtDUYFkKQElRLDHOWN5c96C/view?usp=sharing).

Here is a screenshot of the layer when opened with [Netron](https://github.com/lutzroeder/netron):
![negative paddings 2](https://user-images.githubusercontent.com/42332973/60644510-ec508c80-9e68-11e9-85fe-bf3b1e79548e.png)

Here is the sample code:
```
import onnx
from keras.models import load_model
from keras2onnx import convert_keras

model = load_model('fer2013_mini_XCEPTION.107-0.66.hdf5')
onnx_model = convert_keras(model)
onnx.save(onnx_model, 'minixception_sample.onnx')
```

**Comment by jiafatom** [comment]
@jojaii I have a fix for this, will check in soon.

**Comment by jojaii** [comment]
Thanks @jiafatom. Will wait for the fix.

**Comment by jiafatom** [comment]
> Thanks @jiafatom. Will wait for the fix.

@jojaii code checked in, thanks for reporting the issue.
